### PR TITLE
System Center Endpoint Protection is not supported on Windows Server …

### PR DESCRIPTION
**Pull request by PanuSaukko**
…2012R2

I added for Windows Server 2012 R2 pre-reqs that you should make sure that System Center Endpoint Protection is not installed if you want to install Microsoft Defender for Endpoints. It is not clearly stated.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/configure-server-endpoints.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ For more information, see [Integration with Azure Defender](azure-server-integra
 ### Prerequisites
 
 **Prerequisites for Windows Server 2012 R2** 
-If you have fully updated your machines with the latest [monthly rollup](/troubleshoot/windows-client/deployment/standard-terminology-software-updates.md#monthly-rollup) package, there are **no** additional prerequisites.
+If you have fully updated your machines with the latest [monthly rollup](/troubleshoot/windows-client/deployment/standard-terminology-software-updates.md#monthly-rollup) package, there are **no** additional prerequisites. Make sure that System Center Endpoint Protection is not installed.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-server-endpoints.md]:
```suggestion

If you have fully updated your machines with the latest [monthly rollup](/troubleshoot/windows-client/deployment/standard-terminology-software-updates.md#monthly-rollup) package, there are **no** additional prerequisites. However, make sure that System Center Endpoint Protection is not installed.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PanuSaukko, kindly commit the suggested change. Many thanks.

File: microsoft-365/security/defender-endpoint/configure-server-endpoints.md (hunk 1 above)
Comment (PanuSaukko) [reply microsoft-365/security/defender-endpoint/configure-server-endpoints.md]: I think removing "but" word makes the sentence clearer:

"However, make sure that System Center Endpoint Protection is not installed."

Main thing is to point out not use SCEP! :-)

Comment (JohanFreelancer9) [reply microsoft-365/security/defender-endpoint/configure-server-endpoints.md]: Fully agree with you, my mistake. Thanks for pointing it out. I have changed the suggestion accordingly. Could you please commit now? Thanks so much.

**Pull request merged**